doc/develop/west/release-notes.rst

```diff
@@ -669,7 +669,7 @@ The developer-visible changes to the :ref:`west-apis` are:
 
 West now requires Python 3.6 or later. Additionally, some features may rely on
 Python dictionaries being insertion-ordered; this is only an implementation
-detail in CPython 3.6, but is is part of the language specification as of
+detail in CPython 3.6, but it is part of the language specification as of
```

Comment:
Should we modify release not post-release?

Reply:
good question. I actually refrained from updating the *zephyr* release notes, but this one felt through the cracks. If people have a strong opinion here I am happy to exclude this file from the change. Another way to look at it is due to the nature of west release notes (i.e all ending up in the same file), one could argue that we are fixing this file so that it looks right for *future* releases :)

Reply:
Doesn't it make sense to treat all release notes equally? 

In any case, I don't have any strong opinions.

Reply:
yeah I'd fix those as well, guess you are scripting this change anyway, would not bother going through the hassle of filtering some files out